http/gl3.xml.bak

```diff
@@ -0,0 +1,1887 @@
+<?xml version="1.0"?>
```

Comment:
Nah, if anything I'll put it in .git/info/exclude, which is a user-specific version of .gitignore. But I only have to to make sure the gl3.xml produced by this PR is correct :smile: